extensions/http-curl/tests/HTTPHandlers.h

```diff
@@ -343,4 +345,104 @@ class DeleteTransactionResponder : public CivetHandler {
   std::string response_code;
 };
 
+class HeartbeatHandler : public CivetHandler {
+ public:
+  explicit HeartbeatHandler(bool isSecure)
+      : isSecure(isSecure) {
+  }
+
+  std::string readPost(struct mg_connection *conn) {
+    std::string response;
+    int blockSize = 1024 * sizeof(char), readBytes;
```

Comment:
:suspect: 

Why do we read size of blocksize in case the buffer size is 1024?
My gut feeling says that this should be std::min or someting like that.

Reply:
Not sure why, but makes sense to simplify this. We are just reading 1024 bytes from the connection in each call. The variable blockSize is extraneous.